src/c++/vdo/tests/DmKcopydFake_t1.c

```diff
@@ -86,7 +86,7 @@ static void testDmKcopyd(block_count_t regionSize)
 
   // Verify that the original data has not been touched.
   char *buffer;
-  UDS_ASSERT_SUCCESS(vdo_allocate(VDO_BLOCK_SIZE * regionSize, char,
+  VDO_ASSERT_SUCCESS(vdo_allocate(VDO_BLOCK_SIZE * regionSize, char,
                                   "verification buffer", &buffer));
   VDO_ASSERT_SUCCESS(layer->reader(layer, 0, regionSize, buffer));
   UDS_ASSERT_EQUAL_BYTES(buffer, data, VDO_BLOCK_SIZE * regionSize);
```

Comment:
Are we planning to convert these types of asserts as well?

Reply:
Which type of assert are you talking about here?

Reply:
UDS_ASSERT_EQUAL_BYTES